.github/workflows/check.yml

```diff
@@ -3,6 +3,13 @@ on:
   push:
   pull_request:
 
+env:
+  POSTGRES_USER: postgres
+  POSTGRES_PASSWORD: postgres
+  POSTGRES_DB: nycmesh
+  POSTGRES_HOST: localhost
```

Comment:
why are we adding them? docker-compose will read from `.env`, so you can just do a `cp .env.example .env` then run docker-compose up.
https://docs.docker.com/compose/environment-variables/set-environment-variables/

Reply:
Ahh I see, that's a cleaner way. Thanks!